portfolio/src/main/webapp/index.html

```diff
@@ -149,32 +140,24 @@ <h3> Why Computer Science? </h3>
               around us. Fortunately, since technology continues to rapidly evovle before our very eyes,
```

Comment:
https://screenshot.googleplex.com/gZXBosEons1 is it possible to use the same width here?